frontend/app/routes.json

```
@@ -423,6 +423,14 @@
                   "en": "/:lang/alerts/subscribe",
                   "fr": "/:lang/alertes/abonner"
                 }
+              },
+              {
+                "id": "$lang+/_protected+/alerts+/subscribe+/confirm",
+                "file": "routes/$lang+/_protected+/alerts+/subscribe+/confirm.tsx",
+                "paths": {
+                  "en": "/:lang/alerts/confirm",
+                  "fr": "/:lang/alertes/confirmer"
```

Since are going to have an equivalent page for unsubscribing:

```suggestion
                  "en": "/:lang/alerts/subscribe/confirm",
                  "fr": "/:lang/alertes/abonner/confirmer"
```

---

Done

---

The paths were changed to:
```
"en": "/:lang/subscribe/confirm",
"fr": "/:lang/abonner/confirmer"
```

But we left out the `/alerts`/`/alertes`. The paths should be:
```
"en": "/:lang/alerts/subscribe/confirm",
"fr": "/:lang/alertes/abonner/confirme
```